### ISSUE TITLE
Cannot install from Marketplace, fails with "[Install Error]:release-asset-not-found"

### ISSUE THREAD

**Issue by jwmcglynn** [benjypng/logseq-quicktodo-plugin#3]
When I try to install from the Marketplace, this fails inside the Logseq app, displaying a toast that says:

> [Install Error]:release-asset-not-found

Looking in the console, I don't see the cause, but do see one message appear when I click install:

```
plugin.cljs:142 $cljs_bean$core$Bean$$ {meta: null, obj: {…}, $recursive_QMARK_$: true, $prop__GT_key$: ƒ, $key__GT_prop$: ƒ, …}$__arr$: null$__cnt$: null$__hash$: null$cljs$lang$protocol_mask$partition0$$: 2164131599$cljs$lang$protocol_mask$partition1$$: 139268$key__GT_prop$: ƒ ($x$jscomp$702$$)$prop__GT_key$: ƒ $cljs$core$keyword$$($var_args$jscomp$166$$)$recursive_QMARK_$: truemeta: nullobj: {title: 'Quick Todo', description: "This simple plugin helps you to insert a TODO item…r today's journal page, without navigating to it.", author: 'hkgnp', repo: 'hkgnp/logseq-quicktodo-plugin', icon: './icon.svg', …}[[Prototype]]: Object
(anonymous) @ plugin.cljs:142
emit @ VM14 node:events:394
onMessage @ VM111 renderer_init:81
```

I don't see a plugin version in the Logseq app, but at the time I filed this bug the latest quicktodo plugin release was v1.2.

My Logseq version is 0.5.9.

**Comment by benjypng** [comment]
@jwmcglynn hi!

Hmm, I just tried installing and it seems to be able to do so. Did you install the plugin manually before?

**Comment by jwmcglynn** [comment]
Hmm, I didn't install the plugin manually, this was on an install that previously did not have plugins, and installing them for the first time.

However, I just tried again and can no longer repro.  The original repro was on Linux, and first tried on Mac and it worked.  Then I tried again on the original Linux machine and it worked again.  I'm calling it a fluke, however if this returns one thing that comes to mind is that it failed on Linux, which has a case-sensitive filesystem, where Mac/Windows may still work since their filesystem is case-insensitive.
